code/lib/cli/src/autoblock/block-dependencies-versions.ts

```diff
@@ -0,0 +1,67 @@
+import { createBlocker } from './types';
+import { dedent } from 'ts-dedent';
+import { lt } from 'semver';
+
+const minimalVersionsMap = {
+  'react-scripts': '5.0.0',
+  vue: '3.0.0',
```

Comment:
does storybook 6 support these versions?
if not.. we can't implement these as autoblockers.

Reply:
Can you elaborate on what you mean? If someone tries to upgrade from 6 to 7 or 8, we want to stop the upgrade if we detect unsupported versions. I don't understand the relationship to Storybook 6 tbh.

Reply:
I ask if SB6 supports Angular15 because (assuming it doesn't):

1. The users is on SB6
2. They try to upgrade to SB8
3. We block them, tell them to upgrade to Angular15 first
4. They upgrade to Angular15, their apps breaks, they fix whatever is broken, they check storybook too, and it's broken... They can't figure out how to fix because SB6 does not support Angular15

Assuming the user will check if their storybook still works just before calling the upgrade command again...

Reply:
This is a chicken and egg problem, then. Even if we don't have the auto blocker in place, Storybook will not work in both scenarios. If the user first upgrades to Angular but doesn't upgrade Storybook in parallel, Storybook will break. If they upgrade Storybook but not Angular, Storybook will break as well. The auto blocker doesn't change anything in this situation for upgrades from 6 to 8. The same applies for Next.js. I am not sure about Svelte and Preact, though, whether Storybook supports their latest versions in 6. I am explicitly talking about the upgrade from 6 to 8, since you have mentioned Storybook 6 in your question. 

In the upgrade process of 7 to 8, the auto blocker is also preferred since the user should first upgrade their framework to the latest supported version (which is still supported in Storybook 7) and then upgrade to Storybook 8.

Reply:
@shilman, @vanessayuenn, Can you decide whether we should include auto blockers for Angular, Next.js, Svelte, and Preact, as discussed above?

Reply:
The chicken & egg problem isn't new; it's always been there, and we have been telling users to upgrade stuff in automigrations.

We could move this to autoblockers, but I'm not convinced that will be a net positive for users.